lib/venice/receipt.rb

```diff
@@ -116,6 +119,11 @@ def verify!(data, options = {})
             client = Client.production
             retry
```

Comment:
21007 and 21008 errors will be resolved by the next verify process so we don't need to check `retry_count` here.